e2e/support/component-webpack.config.js

```diff
@@ -81,16 +81,55 @@ module.exports = {
 };
 
 function resolveEmbeddingSdkPackage() {
-  let isEmbeddingSdkPackageInstalled = false;
-
   try {
-    const packagePath = require.resolve("@metabase/embedding-sdk-react");
-    if (packagePath.includes("node_modules")) {
-      isEmbeddingSdkPackageInstalled = true;
+    const nodeModulesPath = path.resolve(__dirname, "../../node_modules");
+
+    const sdkInNodeModulesPath = path.resolve(
+      nodeModulesPath,
+      SDK_PACKAGE_NAME,
+    );
+
+    // Try to resolve the package from the ancestor node_modules folder
+    const hasEmbeddingSdkInNodeModules = fs.existsSync(sdkInNodeModulesPath);
+
+    if (hasEmbeddingSdkInNodeModules) {
+      return {
+        isEmbeddingSdkPackageInstalled: true,
+        embeddingSdkPath: sdkInNodeModulesPath,
+      };
     }
```

Comment:
For some reason, imports to `@metabase/embedding-sdk-react` fails unless we **explicitly** resolve to the `node_modules` path. This ensures that the "module not found: `@metabase/embedding-sdk-react`" issue does not happen again.